### DISCUSSION TITLE
Repository URL changed

### DISCUSSION THREAD

**Discussion by BeckeBauer** [kimai/kimai#3710] [On-Premise: Installation & Updates] [answered]
After executing `git remote set-url origin git@github.com:kimai/kimai.git`, I got the following error:

```
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Any hints (maybe a different e-mail address in the command above)?

**Comment by kevinpapst** [comment]
Show me `cat .git/config`

**Comment by BeckeBauer** [comment]
```
/kimai2$ cat .git/config
[core]
	repositoryformatversion = 0
	filemode = false
	bare = false
	logallrefupdates = true
	symlinks = false
	ignorecase = true
[remote "origin"]
	url = git@github.com:kimai/kimai.git
	fetch = +refs/heads/*:refs/remotes/origin/*
[branch "local"]
	remote = origin
	merge = refs/heads/master
```

**Comment by kevinpapst** [comment, marked as answer]
Ah right, stupid me ... the installation docu says to clone with https://github.com/kimai/kimai.git... 

Can you retry with `git remote set-url origin https://github.com/kimai/kimai.git`

**Comment by BeckeBauer** [comment]
Yip, now it works

Thanks for the quick help...

**Reply by kevinpapst** [reply]
I already updated the blog post. My bad, I am authenticated against the git protocol, which 99% of all users are likely not... and forgot that it is the wrong protocol for almost everyone.
